### PR TITLE
reduce npm bundle size

### PR DESCRIPTION
**Pull request by robertsLando**
Before: 
npm notice package size:  51.0 kB                                 
npm notice unpacked size: 288.9 kB     

After:
npm notice package size:  16.5 kB                                 
npm notice unpacked size: 59.4 kB

### CODE REVIEW

**Comment by robertsLando** [comment]
@mcollina I still havent published aedes @0.42.0 as npm is stucked: https://status.npmjs.org/

File: .npmignore [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+# NPM SPECIFIC
+test
+benchmarks
+examples
+docs
+CODE_OF_CONDUCT.md
+.coveralls.yml
+.github
+
+example.js
```
Comment (mcollina) [inline .npmignore:67]: I prefer to keep all the NPM SPECIFIC.

**Comment by robertsLando** [comment]
Is there any pro/cons?

**Comment by mcollina** [comment]
Essentially it keep tests, docs and examples in the same released bundle. It's a form of docs and a way to solidify the release.

**Comment by robertsLando** [comment]
ok. So I will close this as by default npm ignore already keeps the other things (gitignore)

**Pull request closed without merging**